frontend/components/organisms/QuestionDetail/index.tsx

```diff
@@ -80,6 +84,19 @@ const QuestionDetail = ({
                                 {views_count} {views_count > 1 ? 'times' : 'time'}
                             </span>
                         </div>
+
+                        {team_name !== '' ? (
+                            <div className="flex gap-1">
+                                <span>From Team </span>
+                                <span className="text-gray-500">{team_name}</span>
+                            </div>
+                        ) : (
+                            ''
+                        )}
```

Comment:
I think this can be improved

```suggestion
{team_name !== ' '  && (
   <div className="flex gap-1">
       <span>From Team </span>
       <span className="text-gray-500">{team_name}</span>
   </div>
 )}
```